ext/uri/php_uri.c

```diff
@@ -104,67 +106,6 @@ static HashTable *uri_get_debug_properties(zend_object *object)
 	return result;
 }
 
-PHP_METHOD(Uri_WhatWg_InvalidUrlException, __construct)
-{
-	zend_string *message = NULL;
-	zval *errors = NULL;
-	zend_long code = 0;
-	zval *previous = NULL;
-
-	ZEND_PARSE_PARAMETERS_START(0, 4)
-		Z_PARAM_OPTIONAL
-		Z_PARAM_STR(message)
-		Z_PARAM_ARRAY(errors)
-		Z_PARAM_LONG(code)
-		Z_PARAM_OBJECT_OF_CLASS_OR_NULL(previous, zend_ce_throwable)
-	ZEND_PARSE_PARAMETERS_END();
-
-	if (zend_update_exception_properties(INTERNAL_FUNCTION_PARAM_PASSTHRU, message, code, previous) == FAILURE) {
-		RETURN_THROWS();
-	}
-
-	if (errors == NULL) {
-		zval tmp;
-		ZVAL_EMPTY_ARRAY(&tmp);
-		zend_update_property(uri_whatwg_invalid_url_exception_ce, Z_OBJ_P(ZEND_THIS), ZEND_STRL("errors"), &tmp);
-	} else {
-		zend_update_property(uri_whatwg_invalid_url_exception_ce, Z_OBJ_P(ZEND_THIS), ZEND_STRL("errors"), errors);
-	}
-	if (EG(exception)) {
-		RETURN_THROWS();
-	}
-}
-
-PHP_METHOD(Uri_WhatWg_UrlValidationError, __construct)
-{
-	zend_string *context;
-	zval *type;
-	bool failure;
-
-	ZEND_PARSE_PARAMETERS_START(3, 3)
-		Z_PARAM_STR(context)
-		Z_PARAM_OBJECT_OF_CLASS(type, uri_whatwg_url_validation_error_type_ce)
-		Z_PARAM_BOOL(failure)
-	ZEND_PARSE_PARAMETERS_END();
-
-	zend_update_property_str(uri_whatwg_url_validation_error_ce, Z_OBJ_P(ZEND_THIS), ZEND_STRL("context"), context);
-	if (EG(exception)) {
-		RETURN_THROWS();
-	}
-
-	zend_update_property_ex(uri_whatwg_url_validation_error_ce, Z_OBJ_P(ZEND_THIS), ZSTR_KNOWN(ZEND_STR_TYPE), type);
-	if (EG(exception)) {
-		RETURN_THROWS();
-	}
-
-	zval failure_zv;
-	ZVAL_BOOL(&failure_zv, failure);
-	zend_update_property(uri_whatwg_url_validation_error_ce, Z_OBJ_P(ZEND_THIS), ZEND_STRL("failure"), &failure_zv);
-	if (EG(exception)) {
-		RETURN_THROWS();
-	}
-}
```

Comment:
I don't particularly like that you reordered these. It makes the diff hard to read.

Reply:
Yes. I should have waited a little bit, sorry! A few methods had to be removed due to the added implementation aliases, and that was the point I think when I moved a few methods.